test/test_c_interface.c

```diff
@@ -0,0 +1,152 @@
+/** @file
+    @author Ronald Mclaren
+    @date 2021-02-26
+
+    @brief This file is a quick test of bufr_c_interface_mod.
+*/
+
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "bufr.interface.h"
```

Comment:
How does the bufr.interface.h header file get located during compilation, since that header file is in the src subdirectory, but this test code (test_c_interface.c) is in the test subdirectory?  Are you using the -I option when you compile the test code?  If so, I didn't see where that was specified.

Again, just trying to set up my own test code here ;-)

Reply:
`target_link_libraries` when linking with the library will include the path to the header file.
The header file is installed when the library is installed.